photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala

```diff
@@ -242,6 +251,16 @@ object Params {
             "convention.")
         .foreach(x => params.offHeapIndexMapNumPartitions = x)
 
+      opt[String]("input-columns-names")
+        .text(s"A map of the input columns names, in format: response:label|offset:offset|uid:uid ... ")
+        .foreach(x =>
+          x.split("\\|")
+            .foreach { line =>
+              val Array(key, value) = line.split(":").map(_.trim)
+              params.inputColumnsNames.updated(InputColumnsNames.withName(key), value)
+            }
+        )
+
```

Comment:
Hmm... It may be worth doing so then, since our API is heading towards unity with spark.ml